packages/near-fast-auth-signer/src/components/SignMultichain/SignMultichain.tsx

```diff
@@ -172,34 +218,37 @@ function SignMultichain() {
     }
   };
 
-  if (isDomainKey) {
+  if (isDomainKey && isSafariBrowser) {
     return (
       <ModalSignWrapper ref={signTransactionRef}>
-        <div className="modal-footer">
+        <div className="modal-top">
+          <h3>Passkey Required</h3>
+          <h5>{inFlight ? 'Follow the prompts to continue.' : 'Use your passkey to confirm the transaction'}</h5>
+        </div>
+        <Container>
           <Button
-            variant="affirmative"
+            variant="primary"
             size="large"
-            label={inFlight ? 'Loading...' : 'Approve'}
+            label={inFlight ? 'Loading...' : 'Continue'}
             onClick={onConfirm}
             disabled={
               inFlight || !isValid || typeof amountInfo.price !== 'number'
             }
           />
-        </div>
+        </Container>
         {error && <p className="info-text error">{error}</p>}
       </ModalSignWrapper>
     );
   }
 
   return (
-    <ModalSignWrapper ref={signTransactionRef}>
+    <ModalSignWrapper ref={signTransactionRef} hide={message?.domain === origin ? 'true' : undefined} warning={isUnsafe}>
```

Comment:
Why not a boolean for the hide? 

Then you could do only message.domain === origin

Reply:
yeah, but on console, it was showing warning about mounting boolean on HTML element, with saying that it suggests to apply above solution

Reply:
nvm, I have reverted into boolean

Reply:
Thanks Danny :)